v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/dbutils/dml/CassandraDMLGenerator.java

```diff
@@ -340,8 +347,12 @@ private static Map<String, PreparedStatementValueObject<?>> getColumnValues(
         response.put(colName, columnValue);
         continue;
       }
-      String spannerColumnName =
-          schema.getToSpanner().get(spannerTable.name()).getCols().get(colName);
+      String spannerColumnName = "";
+      try {
+        spannerColumnName = schemaMapper.getSpannerColumnName("", sourceTable.name(), colName);
+      } catch (NoSuchElementException e) {
+        continue;
```

Comment:
please add a logger for the columns which have been skipped

Reply:
The failure case during mapping will throw a different exception bubbling the the very top where as an expected cases skips silently. The failure should get logged at the top layer inside process element.